docs/rest_api.md

```diff
@@ -314,25 +314,28 @@ Response body:
 
 `GET /node/<node>`
 
-Show detailed information about a node. The response includes the
-following fields:
+Show details of a node.
 
-* "name", the name/label of the node.
-* "free", indicates whether the node is free or has been allocated
-    to a project.
-* "nics", a list of nics, each represted by a JSON object having
-    at least the following fields:
-        * "label", the nic's label.
-        * "macaddr", the nic's mac address.
+Returns a JSON object representing a node.
+The object will have at least the following fields:
 
-Response body:
+        * "name", the name/label of the node (string).
+        * "project", indicates whether the node belongs to a project, displaying null if not and the name of the project if so.
```

Comment:
can you revise the sentence. 
Ending a sentence on 'so' does not sound right. 
the meaning is not clear as to when it is null.